Game/ui/inventorymenu.cpp

```diff
@@ -204,11 +205,30 @@ void InventoryMenu::keyDownEvent(KeyEvent &e) {
     else if(sel.sel+1<pg.size())
       sel.sel++;
     }
+  else if(e.key==KeyEvent::K_Z) {
+    lootMode = LootMode::Ten;
+    takeTimer.start(200);
+    onTakeStuff();
+    }
+  else if(e.key==KeyEvent::K_X) {
+    lootMode = LootMode::Hundred;
+    takeTimer.start(200);
+    onTakeStuff();
+    }
+  else if(e.key==KeyEvent::K_Space) {
+    lootMode = LootMode::Stack;
+    takeTimer.start(200);
+    onTakeStuff();
+    }
   adjustScroll();
   update();
   }
 
 void InventoryMenu::keyUpEvent(KeyEvent &e) {
+  if(e.key==KeyEvent::K_Space || e.key==KeyEvent::K_Z || e.key==KeyEvent::K_X){
```

Comment:
Is this check truly necessary?
The keyUpEvent will be called, only if previous keyDownEvent was accepted - it's save to call takeTime.stop() unconditionally